mlflow/utils/environment.py

```diff
@@ -25,10 +25,11 @@ def _mlflow_conda_env(path=None, additional_conda_deps=None, additional_pip_deps
     """
     env = yaml.safe_load(_conda_header)
     env["dependencies"] = ["python={}".format(PYTHON_VERSION)]
+    pip_deps = ["mlflow"] if install_mlflow else [] + additional_pip_deps if additional_pip_deps \
```

Comment:
looks like the binding order doesn't work here. need parentheses around the if statements.

Reply:
thx, good catch